Dockerfile

```diff
@@ -11,13 +11,17 @@ RUN apk update && \
       openjdk${JDK_VER}-jre \
       freetype \
       wine && \
-	  rm -fr /var/cache/apk/*
+        rm -fr /var/cache/apk/*
 
 # Set-up SGDK
 COPY . /sgdk
 ENV GDK=/sgdk
 
+# Create wrappers to execute .exe files using wine
+RUN /sgdk/bin/create-bin-wrappers.sh
```

Comment:
Executing this will write the files *inside* the container, keeping the repository clean